lib/ash/actions/read/read.ex

```diff
@@ -2425,6 +2428,11 @@ defmodule Ash.Actions.Read do
 
       :bypass ->
         {:ok, %{query | tenant: nil, to_tenant: nil}}
+
+      :bypass_all ->
+        query = Ash.Query.set_context(query, %{shared: %{multitenancy: :bypass_all}})
```

Comment:
using shared context means it will apply to all children, regardless of how deep they are nested. Is this the intent? IIRC you made a good point about it being less confusing if it only extends to direct relationships.

Reply:
This way is better 👌